python/lsst/rubintv/analysis/service/database.py

```diff
@@ -386,14 +401,17 @@ def fetch_data(self, query_model: sqlalchemy.Select) -> dict[str, list]:
         return {str(col): [row[i] for row in data] for i, col in enumerate(result.keys())}
 
     def get_column_models(
-        self, columns: list[str]
+        self, columns: list[str], using_aggregator: bool
     ) -> tuple[set[sqlalchemy.Column], set[str], list[sqlalchemy.Column]]:
         """Return the sqlalchemy models for a list of columns.
 
         Parameters
         ----------
         columns :
             The names of the columns in the database.
+        using_aggregator : bool
```

Comment:
For this and other docstring updates in this module: there's no need to include the the type in the docstring since we're using type hints. The documenteer build system should be able to build the docs from the type hints,